flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/FileSystemTableFactory.java

```diff
@@ -78,8 +78,9 @@
 
 	public static final ConfigOption<Long> SINK_ROLLING_POLICY_TIME_INTERVAL = key("sink.rolling-policy.time.interval")
 			.longType()
-			.defaultValue(60L * 1000L)
-			.withDescription("The maximum time duration a part file can stay open before rolling (by default 60 sec).");
```

Comment:
how about "time duration" -> interval time ?
which can keep the option align

Reply:
I write "time duration" to provide an another explaintion to let user better understand it.